src/readme.js

```diff
@@ -43,17 +43,16 @@ const getReadmeTemplate = async templatePath => {
 }
 
 /**
- * Build README content with the given answersContext and templateName
+ * Build README content with the given answersContext, templateName, customTemplate
  *
  * @param {Object} context
  * @param {string} templateName
+ * @param {string} customTemplate
  */
-const buildReadmeContent = async (context, templateName) => {
+const buildReadmeContent = async (context, templateName, customTemplate) => {
```

Comment:
Based on my previous comment, this function would change a bit. It would take two arguments : 

- `context`
- `templatePath` (instead of templateName) that would be the full path to the template